### ISSUE TITLE
Support MBI6030

### ISSUE THREAD

**Issue by karevivan** [FastLED/FastLED#712]
Add support for Macroblock [MBI6030](https://cdn.datasheetspdf.com/pdf-down/M/B/I/MBI6030-Macroblock.pdf).
MBI6030 provides SPI-like interface (CKI, SDI), a two-wire transmission interface, to address the data, so that it receives the data directly without a latch command.
Need provides commands:
1. 16-bit gray scale data;
2. 10-bit gray scale data;
3. 6-bit dot correction data;
4. Frequency GCLK (4.5MHz±10% or 9MHz±10%).

**Comment by focalintent** [comment]
This will likely wait until I finish the RGBW/16-bit support.

**Comment by zackees** [comment]
Deferred by automated triage: New clocked (SPI-like, CKI/SDI) chipset request. Not supported: grep for MBI6030 across src/ returns nothing. Deferred as a design-question feature rather than a mechanical addition, same class as #589 (UCS5603). Precedent exists for clocked chipsets -- src/fl/chipsets/ has apa102.h, hd108.h, lpd880x.h, p9813.h, sm16716.h, ws2801.h -- and HD108 in particular is already a 16-bit clocked part with a gamma LUT (encoders/hd108.h), so the 16-bit grayscale mode has a close template to follow. The open questions are the parts that have no analogue in the existing chipsets: (1) the MBI6030 has TWO grayscale modes, 16-bit and 10-bit, selected by command -- FastLED's clocked chipsets have a single fixed output width, so this needs either two chipset types or a runtime mode; (2) 6-bit dot correction is per-channel calibration data with no equivalent in the CRGB pipeline -- same shape of API question as the UCS5603 current-gain bits on #589, and per CLAUDE.md any new global setter belongs on CFastLED; (3) it needs a GCLK reference clock at 4.5MHz or 9MHz, which is a second output signal FastLED's clocked SPI path has no concept of (APA102-style chipsets use data+clock only) -- that likely needs platform-specific PWM/timer support and is the biggest unknown. focalintent's 2019 note said this would wait on the 16-bit work; that work has since landed (UCS7604 8/16-bit encoders, HD108), so the blocker now is the GCLK and mode/dot-correction API design, not bit depth. Datasheet linked in the issue body.

**Comment by zackees** [comment]
Marking **deferred** — still unsupported (`grep MBI6030 src/` returns nothing), and it needs design decisions rather than just implementation time.

**@focalintent's 2019 reason no longer applies.** He said *"this will likely wait until I finish the RGBW/16-bit support"* — that work has since landed. FastLED now has UCS7604 8/16-bit encoders and **HD108**, which is already a 16-bit clocked part with a gamma LUT (`src/fl/chipsets/encoders/hd108.h`). So the 16-bit grayscale mode has a close template to copy, and bit depth is no longer the blocker.

**What blocks it now** — three things with no analogue in the existing clocked chipsets (`apa102.h`, `hd108.h`, `lpd880x.h`, `p9813.h`, `sm16716.h`, `ws2801.h`):

1. **Two grayscale modes.** 16-bit and 10-bit, selected by command. Every current FastLED clocked chipset has a single fixed output width, so this needs either two chipset types or a runtime mode switch — an API choice, not a coding detail.

2. **6-bit dot correction.** Per-channel calibration data with no place in the `CRGB` pipeline. Same shape of question as the UCS5603's current-gain bits on #589; per `CLAUDE.md`, any new global setter belongs on `CFastLED` rather than a bare `fl::set_*`.

3. **GCLK — the real unknown.** The part needs a free-running reference clock at 4.5 MHz ±10% or 9 MHz ±10%. That's a *third* signal; FastLED's clocked path drives data + clock only and has no concept of an independent reference clock. Supplying it likely means platform-specific PWM/timer output, which makes this substantially more than "add an encoder."

Point 3 is what makes this different from an ordinary new-chipset request, and why I'm not attempting it in a triage pass.

Deferred rather than closed — the request is legitimate and the datasheet is linked in the issue body. Related and similarly parked: #589 (UCS5603).
